.reuse/dep5

```diff
@@ -10,6 +10,9 @@ Source: https://github.com/HEPCloud/decisionengine_modules
 # License: ...
 # SPDX-FileCopyrightText: 2017 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
+Files: src/*.jsonnet src/*/readme src/*.conf src/*.fixture src/*.csv src/*/AWS/scratch src/*/gce_limits_factory_entries.test src/*/data/monitoring.json src/*/data/passwd src/*/data/expected_metric_values.json
+Copyright: 2017 Fermi Research Alliance, LLC
+License: Apache-2.0
```

Comment:
Lines 16 to 20 should be removed I think. Are a duplicate of these (minus the added file)